README.md

```diff
@@ -585,6 +585,7 @@ await descopeClient.management.tenant.configureSettings('my-tenant-id', {
 });
 
 // Generate tenant admin self service link for SSO configuration (valid for 24 hours)
+// ssoId can be provided for a specific sso configuration
```

Comment:
Do you think we should update the readme with this change?

Reply:
yes